examples/selectors/02_eventlistener-sel.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: EventListener
+metadata:
+  name: listener-sel
+  namespace: foo
+spec:
+  serviceAccountName: foo-el-sa
+  namespaceSelector:
+    matchNames:
+    - foo
+    - bar
```

Comment:
If we specify 2 ns here does it mean EL will search in both the namespace for triggers and its not mandatory to have `trigger` in both the namespace am i right (I mean `trigger` can present in any one of the ns)

What happens if no `trigger` exist in both the ns?

Reply:
Yes. If no trigger is present, then nothing happens.

Reply:
We wait for a user to create trigger.